netlify.toml

```diff
@@ -4,7 +4,7 @@ command = "git submodule update --init --remote && npm install && bundle exec je
 publish = "_site"
 
 [build.environment]
-RUBY_VERSION = "2.6.2"
```

Comment:
@howard-e The Netlify build has failed. I guess it expects a known value from this file? https://github.com/rvm/rvm/blob/master/config/known_strings

Reply:
Yep, pinned it to 3.3.3 instead of 3.3. It should build now